Resources/Prototypes/_Goobstation/tags.yml

```
@@ -323,3 +323,8 @@
 - type: Tag
   id: HudMedicalSecurity
 
+- type: Tag
+  id: HudDiagnostic
+
+- type: Tag
+  id: HudMedicalDiagnostic
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)